From 8d2b203fcd11e4b23da72432d46b78f316a8e0dc Mon Sep 17 00:00:00 2001
From: Matyas Selmeci <matyas@cs.wisc.edu>
Date: Fri, 9 Oct 2015 11:56:53 -0500
Subject: [PATCH] Use jspc compiler for tomcat7

EL7 uses tomcat7 so we need to use jspc-compiler-tomcat7 instead of
jspc-compiler-tomcat6.  The groupId for jspc-compiler-tomcat7 is
org.jasic.mojo.jspc instead of org.codehaus.mojo.jspc, so I also change
the groupId for jspc-maven-plugin as well. Also, I hardcode the
version of jspc-maven-plugin, because it needs to match the version
of jspc-compiler-tomcat7.
---
 gums-service/pom.xml | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/gums-service/pom.xml b/gums-service/pom.xml
index 3177cf2..5bccaed 100644
--- a/gums-service/pom.xml
+++ b/gums-service/pom.xml
@@ -37,12 +37,13 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
   </scm>
   <build>
     <finalName>gums</finalName>
     <plugins>
       <plugin>
-        <groupId>org.codehaus.mojo.jspc</groupId>
+        <groupId>org.jasig.mojo.jspc</groupId>
         <artifactId>jspc-maven-plugin</artifactId>
+        <version>2.0.2</version>
         <executions>
           <execution>
             <goals>
               <goal>compile</goal>
             </goals>
@@ -60,13 +61,13 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
             </includes>
           </sources>
         </configuration>
         <dependencies>
           <dependency>
-            <groupId>org.codehaus.mojo.jspc</groupId>
-            <artifactId>jspc-compiler-tomcat6</artifactId>
-            <version>2.0-alpha-3</version>
+            <groupId>org.jasig.mojo.jspc</groupId>
+            <artifactId>jspc-compiler-tomcat7</artifactId>
+            <version>2.0.2</version>
           </dependency>
         </dependencies>
       </plugin>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
-- 
2.6.3

